### DISCUSSION TITLE
Rebar on external Gpu on capable hardware

### DISCUSSION THREAD

**Discussion by Firzo86** [xCuri0/ReBarUEFI#251] [Q&A]
Hi there,
I have an ASUS G713IE , Ryzen 7 4800H with 3050 discrete GPU.
I hooked up a 3070 external gpu through the m.2 nvme slot (PCIe 4x ) and got it to work pretty well.
I've noticed that Resizable bar works for the 3050 mobile gpu but not for the 3070.
I can't seem to get windows to allocate a large memory address for the 3070.
I'm assuming the bios, which is completely locked on my laptop and doesn't even show any option for Rebar, it's not configured to do so on the 4x bus but i can't really find anything on the matter.
any insight would be appreciated.
Cheers!

**Comment by xCuri0** [comment]
If your laptop has working 4G/Rebar already there's no reason ReBarUEFI won't work but obviously flashing modified BIOS is risky and I've heard some laptops have protections like BootGuard so you can't modify it. And you're going to need a hardware flasher for it anyways

If you're lucky you'll find an option to enable Resizable BAR for all devices in the hidden menu, RebarUEFI wiki has a guide on how to see the hidden menu. It should be safe to do as you're not flashing anything and if it goes wrong you can just CMOS reset.
